docs/output.md

```diff
@@ -59,6 +66,61 @@ The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They m
 
 Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQC. The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability. For more information about how to use MultiQC reports, see <http://multiqc.info>.
 
+## Results interpretation
+
+
+A bit more technical background: When concatenating the anndata objects performing an outer join, the missing values will be filled with 0s instead of NaNs. This is because 0 values use nearly no memory in the sparse representation. Filling them with NaNs will make the memory blow up.
+
+The reports generated within MultiQC by itself are not yet providing the ultimate insights to biologically interpret the findings.  Granting the reports some further time to evolve, we suggest to employ [CELLxGENE])(https://cellxgene.cziscience.com/) for now to help with an integration of public data and the interactive generation of publication-ready figures.  If so required, the following files are generated that can be read by statistic suites to select genes/cells of interest:
+
+<details markdown="1">
+<summary>Meant for other tools / own developments</summary>
+| File                                          | What it represents |
+|-----------------------------------------------|--------------------|
+|./adata/merged.h5ad                            | Describes all genes, missing data described as N/As. |
```

Comment:
This is the only output that can be used for other tools. However, missing data is represented as 0s instead of N/As.